core-terms-vdp.md

```diff
@@ -0,0 +1,49 @@
+# [organization] Vulnerability Disclosure Policy
+
+## Introduction
+
+[organization] welcomes feedback from security researchers and the general public to help improve our security. If you believe you have discovered a vulnerability, privacy issue, exposed data, or other security issues in any of our assets, we want to hear from you. This policy outlines steps for reporting vulnerabilities to us, what we expect, what you can expect from us.
+
+## Scope
+
+This policy applies to any digital assets owned, operated, or maintained by [organization], including public facing websites.
+
+## Our Commitments
+
+When working with us, according to this policy, you can expect us to:
+
+- Respond to your report promptly, and work with you to understand and validate your report 
+- Strive to keep you informed about the progress of a vulnerability as it is processed
+- Work to remediate discovered vulnerabilities in a timely manner
+- Extend Safe Harbor for your vulnerability research that is related to this policy
+
+## Our Expectations
+
+In participating in our vulnerability disclosure program in good faith, we ask that you:
+
+- Play by the rules, including following this policy and any other relevant agreements. If there is any inconsistency between this policy and any other applicable terms, the terms of this policy will prevail;
+- Report any vulnerability you’ve discovered promptly;
+- Avoid violating the privacy of others, disrupting our systems, destroying data, and/or harming user experience;
+- Use only the Official Channels to discuss vulnerability information with us;
+- Handle the confidentiality of details of any discovered vulnerabilities according to our Disclosure Policy;
```

Comment:
Just because I could see people copy-and-pasting these terms, and it might not be clear that Disclosure Policy is something separate, thoughts on adding a default disclosure policy of 90 days? This would have the added benefit of driving more organizations to 90 days and showing that it is an industry standard.

Sample language:

> Keep the details of any discovered vulnerabilities confidential until either they are fixed or at least 90 days have passed.

Reply:
@cablej I'd like to work towards this, but am concerned that it creates too much friction for orgs considering whether or not they should do a VDP in the first place (this decision usually comes before a decision to set a proactive timeline). The Disclose.io Status including in this PR introduces a maturity model where Safeharbor plus CVD is the "three gold star - you're the awesomest" variant of a VDP policy to encourage orgs towards CVD... Do you think that's enough, or would providing timelines boilerplates help here as well?

Reply:
@cablej The other challenge which needs to be solved is the varying timeline requirements of products - 90 days is perfectly reasonable imo for SaaS platforms and widely-deployed on-premise software, but mightn't be suitable for a voting machine company, medical device manufacturer, satellite equipment company, car company etc etc etc where the regression testing and in-the-wild patch deployment cycles are potentially longer. Any thoughts on how to address these variations are definitely appreciated - It's an easy conversation to have with organizations 1-to-1 but a difficult one to assert a template on.

Reply:
While I'd like to see faster vuln remediation, a 90-day deadline is not feasible for many situations or organizations. Trying to impose a 90-day deadline on disclose.io users seems to be separate from the goal of broadening VDP adoption.

Reply:
@harleygeiger Perhaps putting different options in there could work as an approach? Alternatively, we leave this as is, make the other repo changes, and kick off a draft for improving the variations on disclosure terms

Reply:
Maybe go the route of CISA's directive and state `Provide us a reasonable amount of time to resolve the issue before you disclose it publicly.`

Reply:
Keep the details of any discovered vulnerabilities confidential _while we are assessing and/or mitigating the vulnerabilities_, until either they are _addressed or an unreasonable amount of time has passed without progress_.